packages/fast-tooling-react/src/navigation/navigation-tree-item.tsx

```diff
@@ -218,7 +219,7 @@ const NavigationTreeItem: React.RefForwardingComponent<
 
 export { NavigationTreeItem };
 
-export const DraggableNavigationTreeItem: typeof NavigationTreeItem = DropTarget(
+export const DraggableNavigationTreeItem = DropTarget(
     NavigationTreeItemDragID,
```

Comment:
linter doesn't spit with no type def?

Reply:
nope, it is implicitly declared with a private type called DndComponentClass